build.rs

```diff
@@ -85,7 +102,7 @@ fn main() {
         .file(haskell_dir.join("parser.c"))
         .compile("tree-sitter-haskell");
 
-    println!("cargo:rerun-if-changed=vendor/tree-sitter-haskell/src/scanner.cc");
+    println!("cargo:rerun-if-changed=vendor/tree-sitter-haskell/src/scanner.c");
```

Comment:
Was causing build cache to be invalidated

Reply:
oh my GOSH, was that it? I've been trying to figure out which of these was doing that and had no luck. Thank you so much!